sigma/plugins/fun/xkcd.py

```diff
@@ -14,37 +14,23 @@
 
     random_comic = (len(args) == 0) #True if args is empty, False otherwise
 
-    if not random_comic:
-        joke_url = 'http://xkcd.com/' + comic_no + '/info.0.json'
-
-        async with aiohttp.ClientSession() as session:
-            async with session.get(joke_url) as data:
-                try: joke_json = await data.json()
-                except Exception:
-                    await message.channel.send(None, embed=discord.Embed(title=':exclamation: Requested xkcd doesn\'t exist', color=0x993333))
-                    return
+    if not random_comic: joke_url = 'http://xkcd.com/' + comic_no + '/info.0.json'
     else:
-        joke_url = 'http://xkcd.com/info.0.json'
-
+        # To grab info of latest comic
         async with aiohttp.ClientSession() as session:
-            async with session.get(joke_url) as data:
-                try: joke_json = await data.json()
-                except Exception:
-                    await message.channel.send(None, embed=discord.Embed(title=':exclamation: Requested xkcd doesn\'t exist', color=0x993333))
-                    return
-        
-        comic_no = str(random.randint(1, joke_json['num']))
-
-        joke_url = 'http://xkcd.com/' + comic_no + '/info.0.json'
-
-        async with aiohttp.ClientSession() as session:
-            async with session.get(joke_url) as data:
-                try: joke_json = await data.json()
-                except Exception:
-                    await message.channel.send(None, embed=discord.Embed(title=':exclamation: Requested xkcd doesn\'t exist', color=0x993333))
-                    return
-
-    embed = discord.Embed(color=0x1abc9c, title='🚽 xkcd Comic #{}: {}'.format(comic_no, joke_json['title']) ).set_image(url=joke_json['img'])
+            async with session.get('http://xkcd.com/info.0.json') as data:
+                joke_json = await data.json()
+        joke_url = 'http://xkcd.com/' + str(random.randint(1, joke_json['num'])) + '/info.0.json'
```

Comment:
a space here would be nice; b/c indent outwards

Reply:
Fixed with 8294c3e